.github/workflows/pr.yml

```diff
@@ -10,35 +10,34 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
```

Comment:
The indentation difference with `build.yml` and `release.yml` got to me.
So this file is indented and has the `$VERSION` var change.